<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>381</y>
     <width>542</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/Tetris.webp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>-230</y>
     <width>482</width>
     <height>422</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 170, 255);
font: 75 20pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(63, 222, 15);
font: 75 20pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
